addons/mail/static/src/core/common/message.js

```diff
@@ -405,10 +405,9 @@ export class Message extends Component {
         ) {
             return;
         }
-        return rpc("/discuss/channel/set_last_seen_message", {
+        return rpc("/discuss/channel/set_new_message_separator", {
             channel_id: message.thread.id,
-            last_message_id: previousMessage ? previousMessage.id : false,
-            allow_older: true,
+            message_id: previousMessage ? previousMessage.id : false,
```

Comment:
We should probably just send `message.id` instead, the python should compute what is the previous message.
It can't be done in JS as we have partial information.

Why not store the current message on the field actually? Wouldn't it be easier?
And instead of using "id > value" we could do "id >= value", it's just a convention.

So instead of "last seen", it would be "first unseen".

Reply:
Or we could keep the same logic but just do id - 1 ?